src/core/iomgr/pollset_multipoller_with_epoll.c

```diff
@@ -194,7 +197,9 @@ static void multipoll_with_epoll_pollset_maybe_work(
     }
     if (pfds[1].revents) {
       do {
+        GRPC_IOMGR_START_BLOCKING_REGION;
```

Comment:
These are unnecessary; this is a 0 timeout poll which is guaranteed to not block.